src/utils/convert-utils.ts

```
@@ -208,6 +177,43 @@ export const getPsmToHlpToCurvePath = async (
     : null;
 };
 
+export type HlpToCurvePath = {
+  hlpToken: string;
+  hlpCurveToken: string;
+  curveToken: string;
+  pool: string;
+  factory: string;
+} | null;
+
+export const getHlpToCurvePath = async (
+  from: string,
+  to: string,
+  network: Network,
+  signerOrProvider: ethers.Signer | ethers.providers.Provider
+): Promise<HlpToCurvePath> => {
+  if (network !== "arbitrum") return null;
```

Instead of returning null, please throw an error here. Seems like the null check is being done on the caller, we want to prevent this.

---

done